build.dunglt.properties

```diff
@@ -0,0 +1,10 @@
+# Managed by Liferay IDE (remove this comment to prevent future updates)
+
+#Managed by Liferay IDE (remove this comment to prevent future updates)
+#Fri Mar 11 13:46:36 ICT 2016
+app.server.tomcat.lib.global.dir = /home/dunglt/work/liferay-portal-6.2-ce-ga6/tomcat-7.0.62/lib/ext
```

Comment:
Kiểu gì mà toàn đường dẫn cứng thế này? chạy ra máy dev khác sao được?
Còn nếu file này không cần thiết thì .ignore luôn.